source/network-security.txt

```diff
@@ -12,18 +12,137 @@ Network Security
    :depth: 1
    :class: onecol
 
-Intro statement
+By default, {+service+} secures your database deployments through the
+use of mandatory TLS/SSL connection encryption, and a VPC-first,
+minimum-access default policy. You can further configure these
+protections to meet your unique security needs and preferences.
 
-{+service+} Features and Best Practices for Network Security
-------------------------------------------------------------
+Use the foundational guidance on this page to plan the network
+security configuration for your {+clusters+}.
 
-Content here
+{+service+} Features and Recommendations for Network Security
+-------------------------------------------------------------
+
+Features
+~~~~~~~~
+
+{+service+} enforces TLS/SSL encryption for all connections to your
+databases.
+
+All {+service+} projects with one or more M10+ dedicated {+clusters+}
+receive their own dedicated VPC (or VNet if you use
+|azure|). {+service+} deploys all dedicated clusters inside this VPC
+or VNet. By default, this VPC or VNet allows no inbound access to
+{+service+}, requiring you to explicitly enable access by one of the
+following methods:
```

Comment:
I would split this sentence in two, and remove the gerund so that our friends the robots can interpret this more easily.

I would also use replacements for VNet and Azure, and to maintain parity with two other cloud providers, I would then add qualifiers for the vpc part (on |aws| and |gcp|).